### PR TITLE
Use queue instead of memory queue in webhook send service

### PR DESCRIPTION
**Pull request by lunny**
Just found webhook send queue is still using a memory one. This PR add that and removed the unused memory queue.

### CODE REVIEW

**Comment by delvh** [comment]
The CI fail sounds related. Somewhere in this PR, there seems to be an NPE.

**Comment by lunny** [comment]
> The CI fail sounds related. Somewhere in this PR, there seems to be an NPE.

Yes, `webhook_service.Init` should be invoked before some tests, blocked by #19393

File: services/webhook/deliver.go [hunk 1]
```diff
@@ -202,17 +202,19 @@ func Deliver(ctx context.Context, t *webhook_model.HookTask) error {
 	return nil
 }
 
-// DeliverHooks checks and delivers undelivered hooks.
+// populateDeliverHooks checks and delivers undelivered hooks.
 // FIXME: graceful: This would likely benefit from either a worker pool with dummy queue
 // or a full queue. Then more hooks could be sent at same time.
```
Comment (Gusted) [inline services/webhook/deliver.go]: I think the comment should be adjusted:
```suggestion
// populateDeliverHooks populates the webhook queue with undelivered webhooks.
```

**Review by Gusted — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Pull request merged**